.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: android
+
+android:
+  components:
+  - tools
+  - platform-tools
+  - build-tools-27.0.3
```

Comment:
should we be using 28?

Reply:
@elihart actually, everything else is on 27 so let's do that separately.